cobigen-templates/templates-devon4j/src/main/templates/crud_ngrx_client_app/templates.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<templatesConfiguration xmlns="http://capgemini.com/devonfw/cobigen/TemplatesConfiguration"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1">
+
+  <templates>
+    <templateExtension ref="${variables.etoName#lower_case}-dialog.component.ts" mergeStrategy="tsmerge"/>
+    <templateExtension ref="${variables.etoName#lower_case}-grid.component.ts" mergeStrategy="tsmerge"/>
```

Comment:
Can you change all the `mergeStrategy` to `override` for TypeScript files? Html or Json can stay as they are :)